pkg/deploy/defaults.go

```diff
@@ -13,41 +13,47 @@
 package deploy
 
 const (
-	DefaultCheServerImageRepo        = "eclipse/che-server"
-	DefaultCodeReadyServerImageRepo  = "registry.redhat.io/codeready-workspaces/server-rhel8"
-	DefaultCheServerImageTag         = "7.0.0-RC-2.0"
-	DefaultCodeReadyServerImageTag   = "1.2"
-	DefaultCheFlavor                 = "che"
-	DefaultChePostgresUser           = "pgche"
-	DefaultChePostgresHostName       = "postgres"
-	DefaultChePostgresPort           = "5432"
-	DefaultChePostgresDb             = "dbche"
-	DefaultPvcStrategy               = "common"
-	DefaultPvcClaimSize              = "1Gi"
-	DefaultIngressStrategy           = "multi-host"
-	DefaultIngressClass              = "nginx"
-	DefaultPluginRegistryUrl         = "https://che-plugin-registry.openshift.io"
-	DefaultUpstreamPluginRegistryUrl = "https://che-plugin-registry.openshift.io/v3"
-	DefaultDevfileRegistryUrl        = "https://che-devfile-registry.openshift.io"
-	DefaultKeycloakAdminUserName     = "admin"
-	DefaultCheLogLevel               = "INFO"
-	DefaultCheDebug                  = "false"
-	DefaultPvcJobsImage              = "registry.redhat.io/ubi8-minimal:8.0-127"
-	DefaultPvcJobsUpstreamImage      = "registry.access.redhat.com/ubi8-minimal:8.0-127"
-	DefaultPostgresImage             = "registry.redhat.io/rhscl/postgresql-96-rhel7:1-40"
-	DefaultPostgresUpstreamImage     = "centos/postgresql-96-centos7:9.6"
-	DefaultKeycloakImage             = "registry.redhat.io/redhat-sso-7/sso73-openshift:1.0-11"
-	DefaultKeycloakUpstreamImage     = "eclipse/che-keycloak:7.0.0-RC-2.0"
-	DefaultJavaOpts                  = "-XX:MaxRAMFraction=2 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 " +
+	DefaultCheServerImageRepo           = "eclipse/che-server"
+	DefaultCodeReadyServerImageRepo     = "registry.redhat.io/codeready-workspaces/server-rhel8"
+	DefaultCheServerImageTag            = "7.0.0-RC-2.0"
```

Comment:
this should be 7.0.0-rc-3.0 (lowercase rc, 3rd release)

Reply:
For now I've not released the `rc-3` release of the operator. And I'd like to be consistent between releases of the underlying docker images and the release of the operator, especially when registries are included inside the deployment. (see my comment [here](https://github.com/eclipse/che-operator/pull/51#discussion_r305358374))